jobs/FunctionTest/FunctionTest.groovy

```diff
@@ -85,7 +84,9 @@ def functionTest(String test_name, String label_name, String TEST_GROUP, Boolean
                         "MANIFEST_FILE=${env.MANIFEST_FILE}",
                         "NODE_NAME=${env.NODE_NAME}",
                         "PYTHON_REPOS=ucs-service",
-                        "TEST_TYPE=$test_type"]
+                        "TEST_TYPE=$test_type",
+                        "KEEP_FAILURE_ENV=${env.KEEP_FAILURE_ENV}",
+                        "KEEP_MINS=${env.KEEP_MINS}"]
```

Comment:
it can be interpreted as keep_minimum or keep_minutes, please make the var name more meaningful.